notebooks/en/issues_in_text_dataset.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # If running on Colab, may want to use GPU (select: Runtime &gt; Change runtime type &gt; Hardware accelerator &gt; GPU)
</pre><p>This you can add to the introduction. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>